Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## v0.6.1 (unreleased)
+- The lower boundary for packets included in ACKs is now derived, and the value sent in STOP_WAITING frames is ignored.
```

Comment:
@lucas-clemente: We probably want to reword this. Suggestions?

Reply:
It's debatable whether it's worth a changelog entry, but SGTM.